index.js

```diff
@@ -1,9 +1,32 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var Funnel = require('broccoli-funnel');
+var MergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
   name: 'torii-provider-arcgis',
+
   isDevelopingAddon: function () {
     return true;
+  },
+
+  included(app) {
+    this._super.included.apply(this, arguments);
+
+    this.import('vendor/arcgis-rest-request/arcgis-rest-request.js');
+    this.import('vendor/shims/arcgis-rest-request/arcgis-rest-request.js');
```

Comment:
I don't see where you've created a shim yet, so for now remove this line.

Reply:
even after dropping the shim.
```
The Broccoli Plugin: [SourceMapConcat: Concat: Vendor /assets/vendor.js] failed with:
Error: ENOENT: no such file or directory, stat '/Users/john6251/github/torii-provider-arcgis/tmp/source_map_concat-input_base_path-CpyoJBuu.tmp/0/vendor/arcgis-rest-request/arcgis-rest-request.js'
```

Reply:
wow. this was because i forgot to append `.umd` to the filename when i imported it.